### PR TITLE
Add HTML Course (changes made)

### PR DESCRIPTION
**Pull request by anubhav-qt**
## What does this PR do?
Add resource(s)

### Description
It is a full course by CodeWithHarry on HTML in Hindi of duratino 2.5 hours.

### Why is this valuable (or not)?
It is valuable because it combines traditional web development fundamentals with modern AI tools, enabling learners to create more dynamic and interactive web applications.

### How do we know it's really free?
YouTube is free!

### For book lists, is it a book? For course lists, is it a course? etc.
It is a course.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -382,6 +382,7 @@
 * [Free Web Development Course For School Students (Grade 5 - 10) \| Certified Course By Coding Blocks Junior \| Learn HTML, CSS, Javascript](https://www.youtube.com/playlist?list=PLhLbJ9UoJCvsCXqP9yAOZpzXHhWkLBXdw) - Coding Blocks Junior
 * [Front End Development Tutorial \| Complete HTML and CSS Tutorial for Beginners (9 Hours)](https://www.youtube.com/watch?v=Eu7G0jV0ImY) - WsCube Tech
 * [HTML CSS & Browser APIs](https://www.youtube.com/playlist?list=PL-Jc9J83PIiHU9RkY9sfh3G64-bd0ptvC) - Pepcoding
+* [HTML Tutorial for Beginners in Hindi | Complete HTML Course with AI + Notes + 5 Project Bundle](https://youtu.be/k2DSi1zGEc8?si=89wK8McUWsGcJnX3) - CodeWithHarry
```
Comment (Thenlie) [inline courses/free-courses-hi.md:385]: For youtube, we prefer "playlist" links and the name of the producer/creator. View [these docs](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md#platforms-and-access-notes) for more information. Once you have updated you PR, you can add “changes made” to the title and reviewer will take another look as soon as they are available. Thanks!

Comment (anubhav-qt) [reply courses/free-courses-hi.md:385]: Thanks for the feedback! The video is structured as a course and is over 2.5 hours long, so, I believe it aligns with the guidelines for single video courses as per the docs. Let me know if there's anything else you'd like me to adjust.

Comment (anubhav-qt) [reply courses/free-courses-hi.md]: I added duration of the video in the file for better clarity.

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -382,6 +382,7 @@
 * [Free Web Development Course For School Students (Grade 5 - 10) \| Certified Course By Coding Blocks Junior \| Learn HTML, CSS, Javascript](https://www.youtube.com/playlist?list=PLhLbJ9UoJCvsCXqP9yAOZpzXHhWkLBXdw) - Coding Blocks Junior
 * [Front End Development Tutorial \| Complete HTML and CSS Tutorial for Beginners (9 Hours)](https://www.youtube.com/watch?v=Eu7G0jV0ImY) - WsCube Tech
 * [HTML CSS & Browser APIs](https://www.youtube.com/playlist?list=PL-Jc9J83PIiHU9RkY9sfh3G64-bd0ptvC) - Pepcoding
+* [HTML Tutorial for Beginners in Hindi | Complete HTML Course with AI + Notes + 5 Project Bundle (2.5 hours)](https://youtu.be/k2DSi1zGEc8?si=89wK8McUWsGcJnX3) - CodeWithHarry
```
[the remark this answers is not in the document]
Comment (anubhav-qt) [reply courses/free-courses-hi.md]: I apologize, I did all the required changes.
Thanks for being patient with me!

File: courses/free-courses-hi.md [hunk 3]
```diff
@@ -382,6 +382,7 @@
 * [Free Web Development Course For School Students (Grade 5 - 10) \| Certified Course By Coding Blocks Junior \| Learn HTML, CSS, Javascript](https://www.youtube.com/playlist?list=PLhLbJ9UoJCvsCXqP9yAOZpzXHhWkLBXdw) - Coding Blocks Junior
 * [Front End Development Tutorial \| Complete HTML and CSS Tutorial for Beginners (9 Hours)](https://www.youtube.com/watch?v=Eu7G0jV0ImY) - WsCube Tech
 * [HTML CSS & Browser APIs](https://www.youtube.com/playlist?list=PL-Jc9J83PIiHU9RkY9sfh3G64-bd0ptvC) - Pepcoding
+* [HTML Tutorial for Beginners in Hindi | Complete HTML Course with AI + Notes + 5 Project Bundle](https://www.youtube.com/watch?v=k2DSi1zGEc8) - CodeWithHarry
```
Comment (eshellman) [inline courses/free-courses-hi.md:385]:
```suggestion
* [HTML Tutorial for Beginners in Hindi \| Complete HTML Course with AI + Notes + 5 Project Bundle](https://www.youtube.com/watch?v=k2DSi1zGEc8) - CodeWithHarry
```
escape pipes

Comment (anubhav-qt) [reply courses/free-courses-hi.md]: Done!

**Pull request merged**